CHANGES.rst

```diff
@@ -5,7 +5,24 @@ Changelog of lizard-auth-client
 1.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Add support for Django 1.7.
```

Comment:
Geldt voor alle toegevoegde changelog entries: verleden tijd is makkelijker leesbaar. "Update the django requirement" klinkt als een instructie die de gebruiker van dit pakket moet doen in plaats van een beschrijving van wat er gedaan is.

Ik weet dat git een andere stijl propageert, maar dat heeft geen invloed op taalkundige correctheid :-)